package.json

```diff
@@ -131,8 +130,6 @@
     "auth-header": "^1.0.0",
     "base64url": "^3.0.1",
     "cross-fetch": "^3.1.4",
-    "events": "^3.3.0",
-    "parse-link-header": "^2.0.0",
```

Comment:
Nice catch! I didn't realise we weren't actually not using these

Reply:
I'm not even sure why they got there in the first place ^^